tremor-script/eqc/spec.erl

```diff
@@ -164,6 +162,44 @@ spec_uop_int(S, N) ->
     ?SHRINK({oneof(['+', '-']), spec_inner_int(S, N - 1)},
 	    [spec_inner_int(S, N - 1)]).
 
+spec_uop_float(S, N) when N =< 1 ->
+    ?SHRINK({oneof(['+', '-']), float_or_float_local(S)},
+	    [float_or_float_local(S)]);
+spec_uop_float(S, N) ->
+    ?SHRINK({oneof(['+', '-']), spec_inner_float(S, N - 1)},
+	    [spec_inner_float(S, N - 1)]).
+
+patch_operation(S, N) ->
+    frequency([{1,
+		{insert, spec_inner_string(S, N - 1),
+		 spec_inner_no_float(S, N - 1)}},
+	       {1,
+		{upsert, spec_inner_string(S, N - 1),
+		 spec_inner_no_float(S, N - 1)}},
+	       {1,
+		{merge, spec_inner_string(S, N - 1),
+		 spec_inner_record(S, N - 1)}},
+	       {1, {erase, spec_inner_string(S, N - 1)}}]).
+
+% Test cases the function spec_uop_record generates
```

Comment:
this documentaiton is in the wrong place

Reply:
Updated, thanks